.env

```diff
@@ -0,0 +1 @@
+export GITHUB_TOKEN=cd07c2a05169db85474a774bfd3eefa153d83c90
```

Comment:
Is the "export" incorrect? Or does the githubingore key have to be the same as the GITHUB_TOKEN?